src/HTML5Backend.js

```diff
@@ -34,25 +34,26 @@ export default class HTML5Backend {
     this.endDragNativeItem = this.endDragNativeItem.bind(this);
   }
 
-  setup() {
-    if (typeof window === 'undefined') {
+  setup(target = window) {
```

Comment:
@darthtrevino Oh, nice! Thanks for that. If I wanted to play around with that, how does one pass along a different context?

e.g. From:

```
export default DragDropContext(HTML5Backend)(IframeDragReceiver)
```

to ... ?